generator/pom.xml

```diff
@@ -69,23 +69,23 @@
 
                             <execution>
                                 <id>Docker login</id>
-                                <phase>deploy</phase>
+                                <phase>validate</phase>
                                 <goals>
                                     <goal>exec</goal>
                                 </goals>
                                 <configuration>
                                     <executable>docker</executable>
                                     <arguments>
                                         <argument>login</argument>
-                                        <argument>--username=${DOCKER_USER}</argument>
-                                        <argument>--password=${DOCKER_PASSWORD}</argument>
+                                        <argument>--username=${env.DOCKER_USERNAME}</argument>
+                                        <argument>--password=${env.DOCKER_PASSWORD}</argument>
                                     </arguments>
                                 </configuration>
                             </execution>
 
                             <execution>
                                 <id>Docker push</id>
-                                <phase>deploy</phase>
+                                <phase>validate</phase>
```

Comment:
It seems to me, this should be "deploy", as I remember deploy step doesn't include validate step, also anyone who would like to execute: `mvnw install` will also deploy artifacts

Reply:
Could you please explain why you decided to use validate instead of deploy?

Reply:
Yes you are right, validate doesn't work. I was trying to prevent the `-pl infrastructure deploy` from updating the JFrog repo, i.e. it should only do the executions in docker profile. Haven't found a good way to do so, maybe I will reach out to you for help today.